CHANGELOG.txt

```diff
@@ -2,6 +2,19 @@
 CHANGELOG
 =========
 
+-------------------------------------------------------------------------------
+Nov, 16, 2021 1.2.1
+-------------------------------------------------------------------------------
+
+major:
```

Comment:
This is probably not a "major" issue. It is basically an input validation.

Reply:
Make sense, will change these to minor.